common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -3,21 +3,46 @@ metadata:
     display_name: Checkboxes
```

Comment:
Do the edits in this file mean that you're changing the default display for all new checkbox (and other types of) problems to include documentation on how to use compound feedback?